ts/components/ConnectionBar.tsx

```diff
@@ -33,6 +33,7 @@ const styles = StyleSheet.create({
     marginRight: customVariables.fontSizeBase / 2
   },
   text: {
+    fontFamily: "RobotoMono-Regular",
```

Comment:
Are you sure this change is also working on iOS? I mean have you checked that on iOS the font is rendered as Roboto-Regular too?
Also why if you want a regular size font that you haven't removed the "fontWeight: "bold" below?

Reply:
Yes I checked and the text seems to be rendered correctly with the RobotoMono font.

![check_font](https://user-images.githubusercontent.com/8174240/56133019-c1524d00-5f8b-11e9-9967-460a89a40edd.png)

I will remove the _fontWeight_ attribute in the next commit.